Sources/PrinceOfVersions/PrinceOfVersions.swift

```diff
@@ -127,19 +130,33 @@ public extension PrinceOfVersions {
         callbackQueue: DispatchQueue = .main,
         notificationFrequency: NotificationType = .always,
         cachePolicy: NSURLRequest.CachePolicy = .reloadIgnoringLocalCacheData,
+        country: String? = nil,
         completion: @escaping AppStoreCompletionBlock
     ) -> URLSessionDataTask? {
 
         guard
-            let bundleIdentifier = bundle.bundleIdentifier,
-            let url = URL(string: "https://itunes.apple.com/lookup?bundleId=\(bundleIdentifier)")
+            let bundleIdentifier = bundle.bundleIdentifier
         else {
             callbackQueue.async {
                 completion(Result.failure(PoVError.invalidJsonData))
             }
             return nil
         }
 
+        var urlString = "https://itunes.apple.com/lookup?bundleId=\(bundleIdentifier)"
+
+        // Append the country parameter if provided
+        if let country = country {
+            urlString += "&country=\(country)"
+        }
```

Comment:
Let's use URL components for this, so do this instead:

```swift
guard
     let bundleIdentifier = bundle.bundleIdentifier,
     var url = URL(string: "https://itunes.apple.com/lookup")
else {
     callbackQueue.async {
          completion(Result.failure(PoVError.invalidJsonData))
    }
     return nil
}

let queryItems: [URLQueryItem] = [
    URLQueryItem(name: "bundleId", value: bundleIdentifier),
    URLQueryItem(name: "country", value: country)
].filter { $0.value != nil }
url.append(queryItems: queryItems)
```

Reply:
I like the suggested change. 
I just changed the `url.append(queryItems: queryItems)` since it is supported from iOS 16.0+.